src/idl_gen_js.cpp

```diff
@@ -361,7 +361,7 @@ static void GenStructBody(const StructDef &struct_def,
 }
 
 // Generate an accessor struct with constructor for a flatbuffers struct.
-void GenStruct(StructDef &struct_def, std::string *code_ptr, std::string *exports_ptr) {
```

Comment:
@gwvo previous commit just removed `parser` from `GenStruct`.  Based on other idl generators, parser is used to tell if its `--gen-mutable` request.  Please correct me if I am wrong.